document/workflow/visual-mapping/visual-mapping.component.ts

```diff
@@ -172,7 +177,23 @@ export class VisualMappingComponent extends FormulaAreaComponent implements OnIn
 
     addAssignment(assignment: CreateAssignmentEvent) {
         const expression = assignment.destination.toXFL() + '=' + assignment.source.toXFL();
-        this.performAction({ type: ModellingActionType.insert, objectId: this.mapping.formulaArea.id, request: this.getInsertRequest(expression ?? this.newFormulaExpression) });
+        this.performAction({ type: ModellingActionType.insert, objectId: this.mapping.formulaArea.id, request: FormulaAreaComponent.getInsertRequest(expression) });
+    }
+
+
+    select(node: SkeletonTreeNode) {
+        this._selectionSubscription?.unsubscribe();
+        if (this.selectedNode) {
+            this.selectedNode.selected = false;
+        }
+        this.selectedNode = node;
+        if (node) {
```

Comment:
use `this.selectedNode` instead of `node` here and below. Maybe `this.selectedNode` is implemented as getter and setter, `node` might differ from it.